src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy;
```

Comment:
better move to util package
```suggestion
package mate.academy.util;
```